src/Umbraco.Configuration/ConfigsFactory.cs

```diff
@@ -61,7 +58,6 @@ public Configs Create(IIOHelper ioHelper, ILogger logger)
             configs.Add<IMemberPasswordConfiguration>(() => MemberPasswordConfigurationSettings);
             configs.Add<IContentSettings>(() => ContentSettings);
 
-            configs.AddCoreConfigs(ioHelper);
```

Comment:
Is it that for the moment we don't add the `GridConfig`?

Reply:
Yes.. `IGridConfig` is not a real config file as the rest.. It is just registered in a normal way.